### PR TITLE
Add environment_files label on service's `env_file` section

### PR DESCRIPTION
**Pull request by ulyssessouza**
**What I did**

**Related issue**
Resolves https://github.com/docker/compose-cli/issues/965

### CODE REVIEW

File: local/compose/labels.go [hunk 1]
```diff
@@ -19,24 +19,26 @@ package compose
...
-
-	//ComposeVersion Compose version
+	containerNumberLabel  = "com.docker.compose.container-number"
+	oneoffLabel           = "com.docker.compose.oneoff"
+	slugLabel             = "com.docker.compose.slug"
+	projectLabel          = compose.ProjectTag
+	volumeLabel           = compose.VolumeTag
+	workingDirLabel       = "com.docker.compose.project.working_dir"
+	configFilesLabel      = "com.docker.compose.project.config_files"
+	environmentFilesLabel = "com.docker.compose.project.environment_files"
```
Comment (ndeloof) [inline local/compose/labels.go:35]: docker-compose label is `LABEL_ENVIRONMENT_FILE = 'com.docker.compose.project.environment_file'` (no `s`)

File: local/compose/create.go [hunk 2]
```diff
@@ -180,6 +180,9 @@ func (s *composeService) getCreateOptions(ctx context.Context, p *types.Project,
 	labels[workingDirLabel] = p.WorkingDir
 	labels[configFilesLabel] = strings.Join(p.ComposeFiles, ",")
 	labels[containerNumberLabel] = strconv.Itoa(number)
+	if len(service.EnvFile) > 0 {
+		labels[environmentFilesLabel] = strings.Join(service.EnvFile, ",")
```
Comment (ndeloof) [inline local/compose/create.go:184]: You're storing the service's env_file attribute in label doing so, while the expected value is the one set by user as `--env-file` on command line (if present)
see 
-> https://github.com/docker/compose/blob/master/compose/cli/command.py#L39
-> https://github.com/docker/compose/blob/master/compose/cli/command.py#L162
-> https://github.com/docker/compose/blob/master/compose/cli/command.py#L177

**Review by ndeloof — CHANGES_REQUESTED** [review]
typo in label name
incorrect value stored

File: local/compose/create.go (hunk 2 above)
Comment (ulyssessouza) [reply local/compose/create.go]: Good catch!

File: cli/cmd/compose/up.go [hunk 3]
```diff
@@ -32,6 +33,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const environmentFileLabel = "com.docker.compose.project.environment_file"
```
Comment (ndeloof) [inline cli/cmd/compose/up.go:36]: why not define this in https://github.com/docker/compose-cli/blob/main/api/compose/tags.go with other labels?

Comment (ulyssessouza) [reply cli/cmd/compose/up.go]: Done

**Review by ndeloof — APPROVED** [review]

**Pull request merged**